app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -116,9 +116,10 @@ private fun WallpaperGroupHeading(
     // and descriptions so they could be localized.
     // In the future, we may want to either use the dynamic wallpaper properties with localized fallbacks
     // or invest in a method of localizing the remote strings themselves.
-    if (collection.name == "classic-firefox") {
+    if (collection.name == "classic-firefox" || collection.name == "default") {
+        val appName = stringResource(R.string.application_name)
```

Comment:
```suggestion
        val appName = stringResource(R.string.app_name)
```
We don't need to define `application_name` since we have different app name depending on whether or not we are Firefox Beta, Firefox Nightly, etc.